modules/troubleshooting-disabling-autoreboot-mco.adoc

```diff
@@ -14,6 +14,7 @@ The following modifications do not trigger a node reboot:
 * changes to the SSH key in the `spec.config.ignition.passwd.users.sshAuthorizedKeys` parameter of a machine config
 * changes to the global pull secret or pull secret in the `openshift-config` namespace
 * changes to the `/etc/containers/registries.conf` file, such as adding or editing an `ImageContentSourcePolicy` object
+* Automatic rotation of the `/etc/kubernetes/kubelet-ca.crt` certificate authority (CA). 
 
 When the MCO detects any of these changes, it drains the corresponding nodes, applies the changes, and uncordons the nodes.
```

Comment:
```suggestion
When the MCO detects any of these changes, it applies the changes, and uncordons the nodes.
```

I believe for some (maybe all?) of these no longer require a drain in 4.8 on. @yuqi-zhang can you confirm?

Reply:
@mike-nguyen @yuqi-zhang The [TE slides](https://docs.google.com/presentation/d/1oQbWtpOH3p4EyNs19DgkQDKcPA4jiuTNKKlKVYT4ixs/edit#slide=id.gcf4412e8e5_1_0) seems to support that there is no drain for cert rotation (and other changes).